.editorconfig

```diff
@@ -496,6 +496,15 @@ dotnet_diagnostic.CA1863.severity = none
 # https://github.com/DotNetAnalyzers/StyleCopAnalyzers
 ##########################################
 
+# SA1005: Single line comments should begin with single space
+dotnet_diagnostic.SA1005.severity = none
+
+# SA1515: Single-line comment should be preceded by blank line
+dotnet_diagnostic.SA1515.severity = none
+
+# SA1512: Single-line comments should not be followed by blank line
+dotnet_diagnostic.SA1512.severity = none
+
```

Comment:
Are these related to the changes? I don't notice any changes that stand out as being obvious matches

Reply:
No, this change just disables some very strict linting rules surrounding comments that were impeding my and Alison's dev loops.

Reply:
IMO if we want strict linting rules like these, we should be automatically applying them at some point, instead of either interrupting flow or breaking compile and forcing engs to fix them one at a time. So painful.